src/bp-templates/bp-nouveau/js/buddypress-messages.js

```diff
@@ -626,7 +626,7 @@ window.bp = window.bp || {};
 						resp.messages[index].id                = value.id;
 						resp.messages[index].content           = value.content;
 						resp.messages[index].sender_id         = value.sender_id;
-						resp.messages[index].sender_name       = value.sender_name;
+						resp.messages[index].sender_name       = $( $.parseHTML( value.sender_name ) ).text();
```

Comment:
@meetmakadiya It's nice to validate that at JS level but we should update this at the PHP level too.

Reply:
@dipskakadiya this value pass to the template so we do not need to validate on PHP side, please share your input and find the attachment.
http://i.imgur.com/G42go19.png

Reply:
@dipskakadiya added validation on PHP side for sender_name

Reply:
@meetmakadiya On frontend side `parseHTML` is not needed anymore